services/httpd/service.go

```diff
@@ -2,38 +2,64 @@ package httpd
 
 import (
 	"crypto/tls"
+	"expvar"
 	"fmt"
 	"log"
 	"net"
 	"net/http"
 	"os"
 	"strings"
+	"sync"
+
+	"github.com/influxdb/influxdb/monitor"
+)
+
+// statistics gathered by the httpd package.
+const (
+	statRequests           = "requests"
```

Comment:
It would good to have some detailed comments about what each of these stats is actually tracking.